### DISCUSSION TITLE
GCP Kubernetes Installation

### DISCUSSION THREAD

**Discussion by fgrande** [Budibase/budibase#9283] [Help] [answered]
Hi all,

I'm quite new to Kubernetes/Helm world, anyway I was able to setup a minikube running installation of BB. Just found a problem with port forwarding, but now is running fine on my machine.

What I'd like to do is to deploy to GKE through Helm. But I'm missing a point....

Where I can find Chart/values files to adapt them to my needs ? I mean.... I already have Redis and Nginx services on my cluster, so I think it would be better to change them (the helm's files) before the deploy. Am I right ? 

Could someone point me in the right direction ? 

Regards
FabioG

**Comment by shogunpurple** [comment, marked as answer]
Hi @fgrande - a few options. 

**Fresh values.yaml**
https://github.com/Budibase/budibase/blob/develop/charts/budibase/values.yaml

**If you want to see the passed in values from a live installation**
You can pull the `values.yaml` directly from a running release (like your minikube setup) as follows:
`helm list` - shows your releases
`helm get values <release name> > values.yaml`

You can then use the `helm upgrade/install` commands with the `-f` flag to pass your `values.yaml` file in to the chart.
